lib/fs.js

```diff
@@ -1585,6 +1585,10 @@ function encodeRealpathResult(result, options, err) {
   }
 }
 
+// This is removed from the fs exports in lib/module.js in order to make
+// sure that this stays internal.
+const realpathCacheKey = fs.realpathCacheKey = Symbol('realpathCacheKey');
```

Comment:
Yeah, I’d follow up with a commit moving this to `internal/fs`. That definitely can’t land in v6.x, though.